src/presentation/http/router/note.test.ts

```diff
@@ -50,3 +50,62 @@ describe('Note API', () => {
     });
   });
 });
+
+describe('Note API', () => {
+  describe('GET note/:notePublicId ', () => {
+    test('Returns note by public id', async () => {
+      const expectedStatus = 200;
+      
+      const expectedNote = {
+        'id': 2,
+        'publicId': 'testnote11',
+        'creatorId': 1,
+        'content': null,
+        'createdAt': '2023-10-16T13:49:19.000Z',
+        'updatedAt': '2023-10-16T13:49:19.000Z',
+      };
+
+      const response = await global.api?.fakeRequest({
+        method: 'GET',
+        url: '/note/testnote11',
```

Comment:
it's better to use realistic mock data. Id should look like real one

Reply:
can you please provide an example of how real id looks like?